corva/models/context.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import annotations
+
+from typing import Any, Optional
+
+import pydantic
+
+
+class CorvaLambdaClientContext(pydantic.BaseModel):
+    @classmethod
+    def from_context(cls, context: Any) -> CorvaLambdaClientContext:
+        return cls(api_key=getattr(context.client_context, 'api_key', None))
+
+    api_key: Optional[str] = None
```

Comment:
what is the purpose of this class?

Reply:
the only reason for it to exist, is to validate the api_key type

Reply:
what kind of validation does this model do? I see that the `api_key` is optional and should be a string. Is this all the validation?

Reply:
yes it is, we can probably delete the model